### ISSUE TITLE
wrong loss penalize？

### ISSUE THREAD

**Issue by ideask** [polarisZhao/PFLD-pytorch#14]
i.e:
#200: 姿态(pose)         0
#201: 表情(expression)   0
#202: 照度(illumination) 0
#203: 化妆(make-up)      0
#204: 遮挡(occlusion)    0
#205: 模糊(blur)         0

the cleaner sample, i.e. without attributes, the loss = 0 ？？？ why???

**Comment by ucohen** [comment]
it seems like the loss function does not match this dataset, as there are numerous items with no attributes - causing zero effect on the loss.
Any suggestions?
